examples/vitess/etcd-controller-template.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: etcd-{{cell}}
+  labels:
+    name: etcd
+    cell: {{cell}}
+spec:
+  replicas: {{replicas}}
+  selector:
+    name: etcd
+    cell: {{cell}}
+  template:
```

Comment:
please set resource limits on this and other pods/controllers so the scheduler can spread them efficiently: https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/user-guide/compute-resources.md